src/arviz_plots/plots/distplot.py

```diff
@@ -250,6 +253,16 @@ def plot_dist(
                 **density_kwargs,
             )
 
+        elif kind == "hist":
+            density = distribution.azstats.histogram(
+                dims=density_dims, **stats_kwargs.get("density", {})
+            )
```

Comment:
we should pass `density=True` as default here. This will normalize the histogram so its area is 1, and will therefore generate histograms with heights similar to those of the kde. That will should take care of the non-reusability of the overlayed example when changing hist/kde. For ecdf/dotplots it will probably break too, but I don't think it matters, ecdf always goes 0-1 whereas dotplots are particularly strange, they should not be expected to act as drop-in replacements of either kde nor hist.

Reply:
Yeah adding that seems to work for the overlay example, without having to alter the `np.linspace` parameters.